lib/pa_ess/http_updater.ex

```diff
@@ -83,7 +83,13 @@ defmodule PaEss.HttpUpdater do
       signs_ui_ms: signs_ui_ms,
       top_line: inspect(top_line),
       bottom_line: inspect(bottom_line),
-      sign_id: sign_id
+      sign_id: sign_id,
+      message_type:
+        case Engine.Config.sign_config(sign_id) do
```

Comment:
This introduces a slight chance of a race condition, if a sign's config changes between when the message is queued and here. That might not be a big deal since this is just used for debugging, but it's worth mentioning. If we want to avoid that, we could do something similar to the audio code path, and add an `extra_logs` value alongside the rest of the queued data, which is fed from the actual data used to generate the message (but that would be slightly more complicated).

If we do decide to go this route, I think something like `current_config` would be a better name than `message_type` for the log key.